core/lib/web3_decl/src/namespaces/zks.rs

```diff
@@ -46,7 +46,10 @@ pub trait ZksNamespace {
     async fn get_bridgehub_contract(&self) -> RpcResult<Option<Address>>;
 
     #[method(name = "getMainContract")]
-    async fn get_main_contract(&self) -> RpcResult<Address>;
+    async fn get_main_l1_contract(&self) -> RpcResult<Address>;
```

Comment:
We now have disconnect between actual RPC method name and method name in core. IMHO it adds more confusion.

Reply:
Yes, but considered the fact, we have more than one main_contract it's useful to have it like this for not messing up. changing public interface is a bad idea